samples/basic/servo_motor/README.rst

```diff
@@ -19,29 +19,22 @@ to modify the source code if you are using a different servomotor.
 Requirements
 ************
 
-You will see this error if you try to build this sample for an unsupported
-board:
+The sample requires a servomotor whose signal pin is connected to a pin driven
+by PWM. The servo must be defined in Devicetree using the ``pwm-servo``
+compatible (part of the sample) and setting its node label to ``servo``. You
+will need to do something like this:
 
-.. code-block:: none
-
-   Unsupported board: pwm-servo devicetree alias is not defined
-
-The sample requires a servomotor whose signal pin is connected to a PWM
-device's channel 0. The PWM device must be configured using the ``pwm-servo``
-:ref:`devicetree <dt-guide>` alias. Usually you will need to set this up via a
-:ref:`devicetree overlay <set-devicetree-overlays>` like so:
-
-.. code-block:: DTS
+.. code-block:: devicetree
 
    / {
-   	aliases {
-   		pwm-servo = &some_pwm_node;
-   	};
+       servo: servo {
+           compatible = "pwm-servo";
+           pwms = <&pwm0 1 20000000 PWM_POLARITY_NORMAL>;
+       };
```

Comment:
The `min-pulse-usec` and `max-pulse-usec` properties are required now, so they should be added also here.

Reply:
added